lib/browser/rpc-server.js

```diff
@@ -280,16 +280,48 @@ const handleRemoteCommand = function (channel, handler) {
   })
 }
 
-handleRemoteCommand('ELECTRON_BROWSER_REQUIRE', function (event, contextId, module) {
-  return valueToMeta(event.sender, contextId, process.mainModule.require(module))
+class CustomEvent {
```

Comment:
I'd rather we use a legit event by making a private method on `WebContents` like `_emit` that will add an event for us.

E.g.

```cpp
bool WebContents::EmitString(std::string s) {
  return Emit(s)
}
```

Then just call that from JS

Reply:
changed as discussed 1:1